src/__tests__/Details.test.js

```diff
@@ -0,0 +1,39 @@
+import { render, waitFor } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import React from 'react';
+import { MemoryRouter } from 'react-router-dom';
+import fetchMock from 'jest-fetch-mock';
+import DetailsPage from '../components/Details';
+import { mockDetails } from './__mocks__/mockCoins';
+
+fetchMock.enableMocks();
+
+describe('DetailsPage', () => {
+  beforeEach(() => {
+    // Mock the fetch request to return the mockCoinData
```

Comment:
- [ ] Great job testing all components. please, you have one failing test. please, kindly fix it.
![Screenshot_14](https://github.com/hamidazim321/coin-watch/assets/65298048/e21d1ac7-26c8-4ff3-a292-73ff37805c3b)
![Screenshot_15](https://github.com/hamidazim321/coin-watch/assets/65298048/6b53c880-64ce-41e8-b744-fcd745b8b668)